Dockerfile

```
@@ -12,7 +12,7 @@ FROM   ubuntu:18.04
 ENV    DEBIAN_FRONTEND noninteractive
 
 # Download and install everything from the repos.
-RUN    apt-get --yes update; apt-get --yes upgrade; apt-get --yes install software-properties-common
+RUN    apt-get --yes update; apt-get --yes upgrade; apt-get --yes install software-properties-common sudo
 RUN    sudo apt-add-repository --yes ppa:webupd8team/java; apt-get --yes update
```

As mentioned, `sudo` should be removed from here as well

```suggestion
RUN    apt-add-repository --yes ppa:webupd8team/java; apt-get --yes update
```